packages/palette-tokens/README.md

```diff
@@ -0,0 +1,46 @@
+# `palette-tokens`
+
+The [design tokens](https://www.lightningdesignsystem.com/design-tokens/) that power [Artsy](https://www.artsy.net)'s [Palette](https://palette.artsy.net).
+
+This package isn't intended to be consumed directly, but rather included as a transitive dependency of palette.
+
+## Usage
+
+In the main theme file...
+
+```
+import buildTheme from "@artsy/palette-tokens"
+export * from "@artsy/palette-tokens";
```

Comment:
It might be nice if rather than needing to scoop this stuff up in a wildcard kind of way we spread it into an object and made things explicit:

```tsx
import { themeValues, buildTheme} from "@artsy/palette-tokens"

const themeConfig = buildTheme(fontFamily);
```

Reply:
There are two things that this exports. The "tokens" which is essentially the themeConfig minus any platform specific things and the few global objects (unitlessBreakpoints, breakpoints) and a bunch of types specified at the bottom of the file. The intent is that the consuming `Theme.tsx` file either in the primary palette package or in eigen should have the same exports as it has today. 

While we want to avoid default exports generally, the default export in this package is important. There's no way to export all named exports _except_ a particular one. We essentially _don't_ want to import the incomplete tokens object in the consuming version of `Theme` because... well... it's incomplete. `themeConfig` is the true representation of that. So, this implementation does that essentially:

```
// only imports the default export
import buildTheme from "@artsy/palette-tokens"
// Exports all _named_ exports from the underlying package
export * from "@artsy/palette-tokens";
```

This is, afaik, the only way to accomplish this goal. Given that we'll only have two files that'll consume this and it _shouldn't_ change, I feel having a default export here is acceptable. 

This documentation is written from the perspective of someone integrating this package into a palette implementation. The export shape is important to maintain compatibility w/ existing palette implementations. I'll update the documentation to reflect this.